IPython/utils/tests/test_text.py

```diff
@@ -180,6 +180,7 @@ def test_LSString():
     lss = text.LSString("abc\ndef")
     nt.assert_equal(lss.l, ['abc', 'def'])
     nt.assert_equal(lss.s, 'abc def')
+    nt.assert_in("path", sys.modules)
```

Comment:
This should check that `lss.get_paths()` works; there's no real significance to checking that 'path' has been imported here.